docs/models-definition.md

```diff
@@ -581,6 +581,25 @@ sequelize.import('project', (sequelize, DataTypes) => {
 })
 ```
 
+This extra capability is available if, for example, ```Error: Cannot find module``` is thrown even though ```/path/to/models/project``` seems to be correct.  Some frameworks, such as Meteor, overload ```require```, and spit out "surprise" results like :
```

Comment:
Inline code in Markdown uses single backticks, three backticks is for code blocks